pkg/webhook/webhook.go

```diff
@@ -556,6 +556,17 @@ func createResourcePatch(patch []jsonPatchOperation, Containers []corev1.Contain
 		patch = patchEmptyResources(patch, 0, "limits")
 	}
 
+	for resourceName, _ := range resourceRequests {
```

Comment:
Just a Nit: golint` for resourceName := range resourceRequests`

Reply:
Done.